examples/configurable/configurable.description

```diff
@@ -4,8 +4,12 @@
 // be applied to a module-level variable declaration that contains an initializer.<br/>
 //
 // The values for `configurable` variables can be provided in a file named
-// `configuration.toml` located in the current directory.
+// `Config.toml`. The file location can be set through an environment variable with
+// the name `BALCONFIGFILE`. If an environment variable is not specified, it is located
+// in the current directory.
+//
 // The configuration file must be in the TOML(v0.4) format.
 // Ballerina currently supports configurable variables of types `int`, `float`,
-// `boolean` and `string`. Future versions will add support for `configurable` variables
+// `boolean`, `string`, `decimal` and the arrays of respective types.
+// Future versions will add support for `configurable` variables
```

Comment:
Do we need this line?

Reply:
ya, I also think we don't need the 'Future versions' line